arangod/Cluster/ClusterFeature.cpp

```diff
@@ -575,11 +580,13 @@ void ClusterFeature::start() {
       << ", server id: '" << myId << "', internal endpoint / address: " << _myEndpoint
       << "', advertised endpoint: " << _myAdvertisedEndpoint << ", role: " << role;
 
-  AgencyCommResult result = comm.getValues("Sync/HeartbeatIntervalMs");
+  auto [acb,idx] = _agencyCache->get(
```

Comment:
What if the agency cache has not yet received any update? In the beginning I would expect that it has to fetch the whole agency store. Since we have only started it above in this method, we might fail here since the data has not yet arrived. This will lead to the server ignoring the value in the agency forever. What can we do? Either we have to wait until the agency cache has its data (could slow down startup), or we have to keep the `getValues` here. Let's discuss.